recipes/graphframes/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "graphframes" %}
+{% set version = "0.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 869aff3f4bf76a609a6bf4399c9c319c9a0280badd676750a49400e940f1adfa
+  patches:
+    - belief_propagation.patch
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - pyspark
+  run:
+    - nose
```

Comment:
Is `nose` really a runtime dependency? That's quite untypical.

Reply:
removed this dependency